src/verify/config.rs

```
@@ -183,14 +185,15 @@ pub fn build_latest_verification_config(config_str: &str) -> Result<LatestVerifi
             } else {
                 value
             };
-            let err = serde_yaml::from_value::<LatestVerificationConfig>(sanitized_value);
-            return Err(anyhow!("Not a valid configuration file: {:?}", err));
+            let err = serde_yaml::from_value::<LatestVerificationConfig>(sanitized_value)
+                .map_err(FetcherErrors::InvalidVerifyFileError);
+            return err;
         }
     };
 
     if config.all_of.is_none() && config.any_of.is_none() {
-        return Err(anyhow!(
-            "config is missing signatures in both allOf and anyOff list"
+        return Err(FetcherErrors::InvalidVerifyFileConfigurationError(
+            "config is missing signatures in both allOf and anyOff list".to_owned(),
```

I think these two error messages can be conflated into a single one, which takes a `String` as parameter. In the 1st case we can return the serde error, in the second one we put the the string we have written.